Dockerfile

```diff
@@ -62,19 +62,17 @@ RUN rm -rf /usr/local/tomcat/webapps/* && \
     mkdir /usr/local/tomcat/webapps/ROOT && \
     chmod +rx /usr/local/bin/docker-entrypoint.sh && \
     mkdir $DHIS2_HOME && \
-    addgroup -S tomcat && \
-    addgroup root tomcat && \
```

Comment:
The `root` user technically does not need to be in any group, but I'm not aware of why this was added in the first place so let me know why it's required and I'll add it back.

Reply:
I don't know why it was added, but agree it probably isn't needed?  @vilkg ?

Reply:
It was easier to manage access this way. If this change works the same way (container is started with non root user with the same read permissions on the same folders), I'm good